README.md

```diff
@@ -75,77 +75,118 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on.
 
-### 0.43 - July 2021 Update
+### 0.45 - August 2021 Update
 
-Our goals for the [v0.43 release cycle](https://github.com/microsoft/PowerToys/issues?q=is%3Aopen+is%3Aissue+project%3Amicrosoft%2FPowerToys%2F22) primarily centered around stability updates and optimizations, installer updates, general bug fixes, and accessibility improvements.
+Our goals for the [v0.45 release cycle](https://github.com/microsoft/PowerToys/issues?q=is%3Aopen+is%3Aissue+project%3Amicrosoft%2FPowerToys%2F23) primarily centered around stability updates and optimizations, installer updates, general bug fixes, and accessibility improvements.
 
-An experimental version of PowerToys (v0.44) will be released the 2nd week of August and will include an updated version of Video Conference Mute.
+We also worked extensively with the community to build an updated settings menu UI. This UI update takes advantage of the latest styling elements to provide users with a refreshed, modern experience navigating the various utilities and their settings.
 
-#### Highlights from v0.43
+An experimental version of PowerToys (v0.46) will be released the 2nd week of September and will include an updated version of Video Conference Mute.
+
+#### Highlights from v0.45
 
 **General**
 
-- New UI for sizes list view in Image Resizer settings. Thanks @niels9001!
-- Fixed FileInUse errors during install/update scenarios.
-- Fixed toggle switches on PowerToys run settings to display correctly.
-- Fixed header text not updating when theme color is changed. Thanks @niels9001!
-- Changed tooltip text for systray icon to be on a single line for Windows 11 compatibility.
-- Expanded the Report Bug tool to collect more robust diagnostic information.
-- Fixed screen reader functionality to stop announcing hidden text in settings.
+- Settings and OOBE windows updated with Fluent UX! We hope you enjoy the new modern feel of the application menus as we align our product with upcoming Windows 11 interfaces. Special thanks to @niels9001 for driving the development of this UI, along with many thanks to the various community members who offered constant feedback and adjustments to make this a truly spectacular update!
+- Added button to settings which displays version history. Thanks @niels9001!
+- Signed PowerToysSetupCustomActions.dll.
+- Improved Update available & Update ready message severity visual. Thanks @niels9001!
+- Improved auto-update experience. Thanks @niels9001!
+- Aligned OOBE theme color with Settings theme color. Thanks @niels9001!
+- Adjusted labeling of "Restart as Administrator" button to "Restart PowerToys as Administrator" to avoid ambiguity in meaning. Thanks @niels9001!
+- Added colored icons to settings sidebar. Thanks @niels9001!
```

Comment:
Obviously, the new style is a huge update and I can't wait to use it. Although @niels9001 is the one who makes this possible in the end, there are many others who actually _opened_ issues. 100% respect for Niels, but please pass some around?

Reply:
That's a fair point, I didn't want to inundate the readme with several callouts on each line, but I tried to capture everyone who contributed meaningfully to the release in the community contributions section, reserving the inline call-out for the members who wrote the code to enable the functionality. 

If I missed anything that a community member did in particular, please let me know so I can accurately give credit to them!

Reply:
Well, of you want a **specific** example... I _did_ choose this point consciously, because I _was_ bugging about the mdl2 icons (and the inconsistency) even before Niels agreed it would be better to use the colored ones. But I don't want it to be about me. And indeed, Niels is the one who done the hard work.